modules/channel-vonage/src/backend/client.ts

```diff
@@ -211,7 +218,10 @@ export class VonageClient {
       await this.sendDropdown(event, payload.options)
     } else if (payload.type === 'carousel') {
       await this.sendCarousel(event, payload)
-    } else if (payload.type === 'image') {
+    } else if (payload.type === 'file') {
+      // We receive payloads of type file but want to send images
+      // FIXME: When we decide to switch to channel renderers
+      payload.type = 'image'
```

Comment:
wait ... if file, payload.type = 'image' ?

Reply:
I'm also confused by this

Reply:
This hack is here to avoid hard-coding the type of payload we are sending to Vonage (see line [142](https://github.com/botpress/botpress/pull/4901/files/b75e51943a7e0d3ddf353b795027a32cb20cb560#diff-d63f2d3924bda7f586a822fd2ae1cd0d13bcc2959cc5979a479380127291abe4R242) where payload.type is used). This will be fixed once we convert this channel to channel renderers